02-molecules/cards/Card/Card.component.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Image from '../../../01-atoms/images/ResponsiveImage/ResponsiveImage.component';
+import { imageTypeProps } from '../../../01-atoms/images/Image/Image.component';
+import bem from '../../../_utils/bem';
+import Heading from '../../../01-atoms/text/Heading.component';
+import Paragraph from '../../../01-atoms/text/Paragraph.component';
+import Button from '../../../01-atoms/buttons/Button.component';
+
+const Card = ({
+  outputImage,
+  image,
+  heading,
+  subheading,
+  body,
+  buttonText,
+  cardModifiers,
+}) => {
+  const block = 'card';
+  return (
+    <div className={bem(block, '', cardModifiers)}>
+      {/* Image */}
+      <div className={bem(block, 'image')}>
+        {image && <Image outputImage={outputImage} image={image} />}
```

Comment:
I just want to double-check. The conditional on line 24 appears inside the card__image wrapper div. Do you always want the parent div to be visible even if there is not an image provided?

Reply:
Nope. Don't want the wrapper if it's empty. I've updated that.